repondeur/zam_repondeur/writer.py

```diff
@@ -36,9 +36,9 @@
     "id_discussion_commune": "Identifiant discussion commune",
     "id_identique": "Identifiant identique",
     "parent": "Parent",
-    "dispositif": "Corps amdt",
-    "objet": "Exposé amdt",
-    "observations": "Objet amdt",
+    "corps": "Corps amdt",
+    "expose": "Exposé amdt",
```

Comment:
Here is the second case where `objet` was previously labeled as `Exposé amdt`, we might have a switch on both side of the chain. It doesn't explain why it looks to be OK on the HTML side though 🤔